Makefile

```diff
@@ -214,66 +213,46 @@ generate-go-apis: ## Alias for .build/generate-go-apis
 		paths=./bootstrap/eks/api/... \
 		paths=./controlplane/eks/api/... \
 		paths=./iam/api/... \
-		crd:crdVersions=v1 \
 		output:crd:dir=config/crd/bases \
-		output:webhook:dir=$(WEBHOOK_ROOT) \
 		object:headerFile=./hack/boilerplate/boilerplate.generatego.txt \
+		crd:crdVersions=v1 \
+		rbac:roleName=manager-role \
 		webhook
 
-	$(CONTROLLER_GEN) \
-		paths=./controllers/... \
-		paths=./$(EXP_DIR)/controllers/... \
-		paths=./bootstrap/eks/controllers/... \
-		paths=./controlplane/eks/controllers/... \
-		output:rbac:dir=$(RBAC_ROOT) \
-		object:headerFile=./hack/boilerplate/boilerplate.generatego.txt \
-		rbac:roleName=manager-role
-
 	$(CONTROLLER_GEN) \
 		paths=./cmd/... \
 		object:headerFile=./hack/boilerplate/boilerplate.generatego.txt
 
 	$(MAKE) defaulters
 
 	$(CONVERSION_GEN) \
-		--input-dirs=./api/v1alpha3 \
-		--input-dirs=./api/v1alpha4 \
+		--input-dirs=./api/v1beta1 \
 		--input-dirs=./cmd/clusterawsadm/api/bootstrap/v1alpha1 \
```

Comment:
I think this folder was also removed, not? Or is that bootstrap/eks... 🤔

Reply:
Are you talking about `./cmd/clusterawsadm/api/bootstrap/v1alpha1 `? It is still there.

Reply:
Yeah, what I was looking at was bootstrap/eks indeed. That's the one that is gone. This is fine. :)